src/lib/detector.ts

```diff
@@ -16,47 +16,11 @@
 
 /**
  * @file detector.ts
- * @author Hugo Masclet <@Hugoo>
- * @author Felix Hildebrandt <@fhildeb>
```

Comment:
??

Reply:
The file was created specifically for the supportsInterface. All other code comes from Yamen (permissions)